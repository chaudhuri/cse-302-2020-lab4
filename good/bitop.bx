// bit operations

proc main() {
  var x = 10843 : int;
  var y = 13394 : int;
  print(x | y);
  print(x & y);
  print(x ^ y);
  print(~ x);
  print(~ y);
  print(x + ~ x);
  print(x >> 3);
  print(x << 3);
  print(-x >> 3);
  print(-x << 3);
}