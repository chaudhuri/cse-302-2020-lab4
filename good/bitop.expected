15963
8274
7689
-10844
-13395
-1
1355
86744
-1356
-86744
